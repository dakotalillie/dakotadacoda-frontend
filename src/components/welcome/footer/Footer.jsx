import React from 'react';
import { Col, Container, Row } from 'reactstrap';

import githubIcon from '../../../img/social_icons/white/github.png';
import mediumIcon from '../../../img/social_icons/white/medium.png';
import linkedinIcon from '../../../img/social_icons/white/linkedin.png';
import links from '../../../utils/links.json';
import './footer.scss';

export default function Footer() {
  return (
    <div className="footer">
      <Container>
        <Row>
          <Col sm="6">
            <span>&copy; Dakota Lillie 2018</span>
          </Col>
          <Col sm="6" className="second-col">
            <a href={links.github}>
              <img src={githubIcon} className="social-icon" alt="Github" />
            </a>
            <a href={links.medium}>
              <img src={mediumIcon} className="social-icon" alt="Medium" />
            </a>
            <a href={links.linkedin}>
              <img
                src={linkedinIcon}
                className="social-icon"
                id="linkedin"
                alt="Linked In"
              />
            </a>
          </Col>
        </Row>
      </Container>
    </div>
  );
}
