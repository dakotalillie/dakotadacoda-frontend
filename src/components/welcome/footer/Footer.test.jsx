import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Footer from './Footer';

describe('Footer', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('renders successfully', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});