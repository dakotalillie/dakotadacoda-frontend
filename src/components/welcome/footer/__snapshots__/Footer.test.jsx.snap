// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders successfully 1`] = `
<div
  className="footer"
>
  <Container
    tag="div"
  >
    <Row
      tag="div"
    >
      <Col
        sm="6"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <span>
          Â© Dakota Lillie 2018
        </span>
      </Col>
      <Col
        className="second-col"
        sm="6"
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <a
          href="https://github.com/dakotalillie"
        >
          <img
            alt="Github"
            className="social-icon"
            src="github.png"
          />
        </a>
        <a
          href="https://medium.com/@dakota.lillie"
        >
          <img
            alt="Medium"
            className="social-icon"
            src="medium.png"
          />
        </a>
        <a
          href="https://www.linkedin.com/in/dakotalillie/"
        >
          <img
            alt="Linked In"
            className="social-icon"
            id="linkedin"
            src="linkedin.png"
          />
        </a>
      </Col>
    </Row>
  </Container>
</div>
`;
